Detections/ThreatIntelligenceIndicator/IPEntity_AzureSQL.yaml

```diff
@@ -0,0 +1,52 @@
+id: d0aa8969-1bbe-4da3-9e76-09e5f67c9d85
+name: TI map IP entity to Azure SQL Security Audit Events
+description: |
+  'Identifies a match in SQLSecurityAuditEvents from any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
```

Comment:
We should add the connector for Azure Key Vault Logs here as well, or do we not have one?